MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
+   fetch('https://ghibliapi.herokuapp.com/films') //we want the data here before user interacton
+   .then(response => response.json())
+
+   .then(dataObj => {
+      fetchMovie(dataObj)
+
+      // fetchMovie(response)
+   })
+
+   // .catch(err =>{
+   //    console.log("Nah Dawg", err)
+   // })
+
+   eventListeners()
+})
+
+var iDontBelong = [null] //API response 
+ //common endpoint
+function fetchMovie(film){  //code initially existed at the end
+   let button3 = document.querySelector('#sumnToWatch')
```

Comment:
Select is not a button and it confused me a bit. Naming your dom element that is easily recognizable is important (Same for js variables for future reference).